dll/cpl/sysdm/lang/en-US.rc

```diff
@@ -278,17 +278,36 @@ BEGIN
     PUSHBUTTON "Cancel", IDCANCEL, 195, 224, 50, 14
 END
 
-IDD_EDIT_VARIABLE DIALOGEX 10, 15, 227, 71
+IDD_EDIT_VARIABLE DIALOGEX 10, 15, 431, 71
```

Comment:
You edited this dialog only in the english version?

Reply:
I completely forgot to edit the dialog for other languages.